### DISCUSSION TITLE
Is Azure supported?

### DISCUSSION THREAD

**Discussion by jghal** [Pluralith/pluralith-cli#103] [Q&A] [answered]
I couldn't find anything one way or the other, thought I'd ask before I try.

**Comment by DanThePutzer** [comment, marked as answer]
Hi @jghal 

Generally yes, we do support Azure, although it might not be as fleshed out as AWS.

If you run into problems or the diagram looks off please do let us know and we'll look into it!

**Reply by jghal** [reply]
thanks
